OCS/deploy-ocs.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+export namespace="openshift-storage"
+export rook_override="${rook_override:-false}"
+export public_network="${public_network:-1.1.1.1/24}"
+export cluster_network="${cluster_network:-1.1.1.1/24}"
+
+export rook_cluster=${namespace}
+
+oc patch featureGate cluster --type merge -p '{"spec":{"featureSet":"TechPreviewNoUpgrade"}}'
+
+if [ ! -d rook ]; then
+    git clone https://github.com/rook/rook
+fi
+
+pushd .
+cd rook/cluster/examples/kubernetes/ceph
+
+if [ ! -f common-modified.yaml ]; then
+    sed -e "s/name: rook-ceph$/name: ${namespace}/" common.yaml > common-modified.yaml
+    sed -ie "s/namespace: rook-ceph/namespace: ${namespace}/" common-modified.yaml
+fi
+
+oc create -f common-modified.yaml
+oc label namespace ${namespace} "openshift.io/cluster-monitoring=true"
+
+if [ ! -f operator-openshift-modified.yaml ]; then
+    sed -e "s/namespace: rook-ceph/namespace: ${namespace}/" \
+        operator-openshift.yaml > operator-openshift-modified.yaml
+    sed -ie "s/:rook-ceph:/:${namespace}:/" operator-openshift-modified.yaml
+fi
+
+oc create -f operator-openshift-modified.yaml
+sleep 20
+
+oc wait --for condition=ready pod -l app=rook-ceph-operator -n ${namespace} --timeout=2400s
+sleep 20
+oc wait --for condition=ready pod -l app=rook-discover -n ${namespace} --timeout=2400s
+
+if [ "$rook_override" == "true" ] ; then
+  envsubst < ~1/rook-config-override.yaml | oc create -f -
```

Comment:
Not a fan of obscurities like `~1` -  a simple variable with the directory would be far more clear

Reply:
removed